tests/Framework/TestCase.php

```diff
@@ -85,9 +86,11 @@ public function getMockHttpResponse($path, $substitutions = array())
      * then the function will replace all instances of '[NAME]' in the
      * response with 'Fake Name'. Substitutions are not required.
      *
-     * @param array<string>|string $paths
+     * @param array<string, string>|string $paths
      * @param array<string, string> $substitutions
      *
+     * @psalm-suppress MixedInferredReturnType
+     * @psalm-suppress MixedMethodCall
```

Comment:
why do these two need to be suppressed? and if they do, can you suppress them just for the one line? (new feature added to psalm since this library was written, which is why most places do it in the docblock)